### DISCUSSION TITLE
Video Playback Speed Adjustment

### DISCUSSION THREAD

**Discussion by Wjxfi** [ente-io/ente#4138] [Enhancements]
Dear Ente Team,

I am addressing you as a loyal user of your storage, and I want to share my experience, which unfortunately leaves much to be desired. The video playback speed below 2.5 proves to be too slow most of the time, and this is causing significant difficulties in my learning. In order to stay productive, I have to download videos for local playback, which is not only frustrating, but also makes me feel like I'm not getting what I paid for.

I urge you to consider adding a playback speed slider that would allow users to adjust the speed from 0.5 to 3. This is not just an improvement - it's a necessity that could make a significant difference in the user experience. Fast and efficient learning is not just a preference, but a vital aspect in our fast-paced world. 

Imagine how many users like me will feel ignored if this feature is not implemented. In a highly competitive market, ignoring the needs of users can cause them to start looking for alternative solutions that will offer them what they want. This can be a disaster for your company, and I'm sure you don't want to lose loyal customers who believed in your product.

I sincerely hope you will take this suggestion into consideration. Your ability to adapt to the needs of your users will not only enhance your reputation, but will also help you retain the trust and loyalty of your customers. Don't let this chance to improve your product and make it truly outstanding pass you by.

Regards,  
Wjxfi

**Comment by Flummic** [comment]
I am a new user an really love Ente so far! I would also love to have easy Playback-Speed-Adjustments available. Especially when training sports I often want to slow down a video to analyze my form/explain something to my students. Here this would really come in handy. 

A nice implementation I saw on Samsung's Gallery was that they automatically slowed the video playback down when you press and hold the screen while a video is playing. But a more flexible control (as suggested in the original post) would probably be even nicer :) 

Note that I am not talking abou permanently setting the speed for all videos or "editin" a video to be slower/faster, but rather I am talking about quick controls while viewing a video in the Gallery :)

**Comment by fosszil** [comment]
Yes, I ran into the same issue as well. Adding playback is definitely needed in the video player. I'd also like a press and hold for 2x playback, once we get the speed controls done. Happy to contribute code to make this happen if approved
